import_logs.py

```diff
@@ -1042,7 +1046,7 @@ def _get_token_auth(self):
                     '../../misc/cron/updatetoken.php'),
             )
 
-            phpBinary = 'php'
+            phpBinary = config.options.php_binary
```

Comment:
might be useful to not do the "windows-only" check some lines below if the `phpBinary` option differs from default. Otherwise it might happen that the script fatals if the `where php.exe` does not return anything but `phpBinary` option was actually set correct...